README.md

```diff
@@ -102,7 +102,7 @@ Zend Engine v2.5.0, Copyright (c) 1998-2015 Zend Technologies
 
 If you want to run composer:
 
-set ```COMPOSER_VERSION``` to the mayor version you want to use (default ```COMPOSER_VERSION=1``` )
+set ```COMPOSER_VERSION``` to the mayor version you want to use (default ```COMPOSER_VERSION=2``` )
```

Comment:
```suggestion
set ```COMPOSER_VERSION``` to the major version you want to use (default ```COMPOSER_VERSION=2``` )
```

can you please fix this type across all PR's? Thanks